client/src/hooks/http.hook.js

```diff
@@ -13,6 +13,12 @@ export const useHttp = () => {
         headers['Content-Type'] = 'application/json';
```

Comment:
I wouldn't recommend creating an API service as a hook, you're limited to use it only inside components.